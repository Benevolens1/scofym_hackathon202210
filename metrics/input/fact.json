{
	"1": {
		"type": "fact",
		"arguments": {"n": 10}
	},
	"2": {
		"type": "fact",
		"arguments": {"n": -3}
	},
    "3": {
		"type": "fact",
		"arguments": {"n": 1307674368000}
	}
}