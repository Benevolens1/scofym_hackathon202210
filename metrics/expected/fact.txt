1 3628800
2 0
3 1307674368000

